src/main/java/com/ita/if103java/ims/dao/impl/DashboardDaoImpl.java

```diff
@@ -102,50 +102,50 @@ public WarehousePremiumStructDto getPreLoadByAccounId(Long id, Long accountId) {
             WarehousePremiumStructDto wpld = jdbcTemplate.queryForObject(Queries.SQL_WAREHOUSE_STRUCTURE_PRIMARY,
                 warehousePremiumStructRowMapper,id, accountId);
             wpld.setLevel(0);
-            ChargeCapacity cc = jdbcTemplate.queryForObject(Queries.SQL_FIND_SUBS_LOAD,
-                chargeCapacityRowMapper,
-                wpld.getId(), accountId, accountId, accountId, accountId);
-            wpld.setCapacity(cc.getCapacity());
-            wpld.setCharge(cc.getCharge());
-            wpld.setChilds(findSub(id,wpld.getLevel(), accountId));
+
+            wpld.setChilds(findSub(id, id, wpld.getLevel(), accountId));
+
+            recFill(wpld);
+
             return wpld;
         } catch (DataAccessException e) {
-            throw crudException(e.toString(),"getPreLoad","accountId=" + accountId);
+            throw crudException(e,"getPreLoad","accountId=" + accountId);
         }
     }
 
-    private List<WarehousePremiumStructDto> findSub(Long id, int level, Long accountId){
+    private List<WarehousePremiumStructDto> findSub(Long id, Long top_warehouse_id, int level, Long accountId){
```

Comment:
top_warehouse_id -please rename to follow Java code style

findSub - please do not shorten methods and variables, because your code is hard to read and understand